### PR TITLE
feat(web): Add ChatPanel to organization pages

### PR DESCRIPTION
**Pull request by Ingimarsson**
# Add ChatPanel to organization pages

## What

This adds the boost.ai chat bot to organization pages.

The ChatPanel component has been extended to support multiple endpoints. The endpoint is written to `window.boostEndpoint` so that when a page with a different endpoint is opened, the old chat panel is removed and a new one initialized.

## Why

By request.

## Screenshots / Gifs

image

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by ingolfurs — APPROVED** [review]
LGTM

**Review by stjanilofts — APPROVED** [review]
LGTM

File: apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx [hunk 1]
```diff
@@ -65,6 +70,20 @@ const OrganizationFooter: React.FC<HeaderProps> = ({ organizationPage }) => {
...
+    return null
+  }
+
+  switch (slug) {
+    case 'syslumenn':
+      return <ChatPanel endpoint="syslumenn" />
+    default:
+      return null
+  }
+}
```
Comment (stefanhar) [inline apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx:85]: Consider making the `slug === endpointKey` so `endpoints[slug]` resolves to correct config, we might not need this abstraction in if that is the case

Comment (Ingimarsson) [reply apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx:85]: I tried doing something like this
```
slug in endpoints ? ​<ChatPanel endpoint={slug} /> : null
```
But TS kept complaining, so I did it this way.

**Pull request merged**